### ISSUE TITLE
Command line tool broken, outputs empty undefined directory when beautifying

### ISSUE THREAD

**Issue by ustun** [prettydiff/prettydiff#252]
I have the following file:

```js
              console.log("Hello world");
```

I issue the following command:
```bash
prettydiff   source:"foo.js" mode:"beautify" output:"."
```

The output is printed on screen, but not written to file. Results in  an empty `undefined` folder.
```
prettydiff   source:"foo.js" mode:"beautify" output:"foo"
```
Results in a folder foo with a subfolder undefined.
```
 prettydiff --version
Version: 1.16.11 Dated: 19 January 2016

```

The only workaround is to use the normal shell redirection.

**Comment by ustun** [comment]
I installed the previous version:
```
npm install prettydiff@1.15 -g
```

This one doesn't even output anything, it just outputs the usage. 

```
 prettydiff source:"foo.js" readmethod:"filescreen" mode:"beautify"

**Comment by prettydiff** [comment]
I can reproduce the defects.  I am looking into this now.

On a side note, even though all of this is clearly a defect and the printing to screen is an unintended behavior.  I will add print to screen if the output is "." as an enhancement while fixing this defect.

**Comment by ustun** [comment]
Thanks a lot! Much appreciated.

**Comment by prettydiff** [comment]
I believe have the issue resolved with this code: http://prettydiff.com/api/node-local.js  You can test this by overwrite the same file in your installed package.  I will publish the update very soon.

Please let me know if you continue to have problems and I will provide additional resolutions.

**Comment by prettydiff** [comment]
Published as v1.16.12
